e2e-tests/tests/conftest.py

```diff
@@ -116,7 +117,16 @@ def pytest_configure(config: Config):
 def pytest_sessionstart(session):
     # set partner chain status on main thread
     if not hasattr(session.config, 'workerinput'):
-        session.config.partner_chain_status = partner_chain_rpc_api.partner_chain_get_status().result
+        # For standard Substrate nodes, create a mock status
+        try:
+            session.config.partner_chain_status = partner_chain_rpc_api.partner_chain_get_status().result
+        except Exception as e:
+            logger.warning(f"Failed to get partner chain status, using mock data: {e}")
+            # Mock status for standard Substrate nodes
+            session.config.partner_chain_status = {
```

Comment:
Also can be moved into PR with enhancements

Reply:
Hey Alex, this is another case where I need this conditional for tacheon tests because there's no partner-chain rpc in this case, we need to use substrate rpc api.

Maybe I'm misunderstanding something here.